code/controllers/subsystem/server_maint.dm

```diff
@@ -88,14 +89,22 @@ SUBSYSTEM_DEF(server_maint)
 	if(tgsversion)
 		SSblackbox.record_feedback("text", "server_tools", 1, tgsversion.raw_parameter)
 
-
+/**
+  * Update our hub status
+  *
+  * This toggles the server between being on and off the hub according to the following things
+  * * If the panic bunker is on, we go off the hub
+  * * If the clients are greater than the maximum hub population config, we go off the hub
+  * * Otherwise we go on the hub
+  *
+  * This is called in Client New and Destroy and in the panic bunker toggle verb
+  */
 /datum/controller/subsystem/server_maint/proc/UpdateHubStatus()
-	if(!CONFIG_GET(flag/hub) || !CONFIG_GET(number/max_hub_pop))
```

Comment:
Any reason to remove the `flag/hub` check?

Reply:
I need to readd it